choices/choices.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!--Importing Google Icon Font-->
```

Comment:
this is a little over commented, the code adequately describes what you are doing and the comments make it slightly harder to read